scripts/verify-status.sh

```diff
@@ -3,42 +3,56 @@
 echo "Checking status of POST Jobs for Eventing-Manager"
 
 REF_NAME="${1:-"main"}"
-TIMEOUT_TIME="${2:-120}"
+TIMEOUT_TIME="${2:-240}"
```

Comment:
I think it is fine if we even increase this to 300 or even 600.

Reply:
Sure